osquery/logger/plugins/aws_util.cpp

```diff
@@ -66,7 +66,12 @@ static const std::set<std::string> kAwsRegions = {"us-east-1",
                                                   "ap-southeast-2",
                                                   "ap-northeast-1",
                                                   "ap-northeast-2",
-                                                  "sa-east-1"};
+                                                  "sa-east-1",
+                                                  "ap-south-1",
+			                          "us-east-2",
```

Comment:
Ah, one last change needed, you missed one space here :)

After you update, if the code audit test passes it will be ready to go.